Makefile

```
@@ -9,41 +9,38 @@ vap_test_files:=\
+		curl https://raw.githubusercontent.com/kubescape/cel-admission-library/refs/heads/main/test-resources/$$word >> public/vap-test-files.yaml; \
+		echo $$word >> public/vap-test-files-index.yaml; \
+	done;
 
 download: wasm-download kubescape-download controls-download
 
-build: 
-	GOOS=js GOARCH=wasm go -C go build -ldflags="-s -w" -o ../dist/main.wasm cmd/main.go 
+build:
+	GOOS=js GOARCH=wasm go -C go build -ldflags="-s -w" -o ../dist/main.wasm cmd/main.go
 
 local: build
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)